Sketch.java

```diff
@@ -24,13 +26,74 @@ public void setup() {
    */
   public void draw() {
 	  
-	// sample code, delete this stuff
-    stroke(128);
-    line(150, 25, 270, 350);  
+	drawFace(200,40);
```

Comment:
as per [level 2 requirement](https://github.com/SACHSTech/ProcessingTask6-CreatingMethods#level-2), use a method to draw an object at various locations specified by the method parameters.  i.e multiple calls to `drawFace()` method